frontend/app/assets/javascripts/spree/frontend/account.js

```diff
@@ -0,0 +1,12 @@
+Spree.fetchAccount = function () {
+  return $.ajax({
+    url: Spree.pathFor('account_link')
+  }).done(function (data) {
+    Spree.accountFetched = true
```

Comment:
Maybe we could initialize all `*Fetched` attributes with `false` somewhere? The code will be easier to understand.